### PR TITLE
Blogpost

### PR DESCRIPTION
**Pull request by happimaker**
feature:Blog post

### CODE REVIEW

File: src/assets/css/style.scss [hunk 1]
```diff
@@ -138,7 +138,7 @@ header {
   
     & h1 {
       font-size: 80px;
-      font-family: 'raleway-ebold';
+      font-family: 'raleway-ebold', sans-serif;
       margin: 0;
       line-height: 90px;
```
Comment (rogue707Dev) [inline src/assets/css/style.scss]: Make this as mixin

File: src/assets/css/style.scss [hunk 2]
```diff
@@ -197,10 +197,18 @@ header {
...
   color: $greenColor;
 }
 
+.green4 {
+  color: $greenColor4;
+}
+
 .melon {
   color: $melonColor;
 }
```
Comment (rogue707Dev) [inline src/assets/css/style.scss]: Put color styles in individual file.

File: src/assets/css/style.scss [hunk 3]
```diff
@@ -213,6 +221,10 @@ header {
   color: $lightNavyColor;
 }
 
+.light-magenta {
+  color: $lightMagentaColor;
+}
+
 @font-face {
   font-family: 'nexa-bold';
   src: url(../fonts/nexa-bold.otf);
```
Comment (rogue707Dev) [inline src/assets/css/style.scss]: Put font style in individual file

File: src/assets/css/style.scss [hunk 4]
```diff
@@ -250,6 +267,10 @@ header {
   font-family: 'OpenSans-BoldItalic';
 }
 
+.opensans-lightitalic {
+  font-family: 'OpenSans-LightItalic';
```
Comment (rogue707Dev) [inline src/assets/css/style.scss]: Use weight as variable.

File: src/index.html [hunk 5]
```diff
@@ -658,5 +658,134 @@ <h3 class="raleway-light black7 flex-center center fontsize-16">
...
+          </div>
+        </article>
+        <div class="blogs">
+          <div class="blog-line">
+            <div class="blog-high-left background-gray2">
+            </div>
+            <div class="blog-high-right background-green2">
+              <p class="fontsize-12 raleway-regular">24/12/2014</p>
+              <p class="fontsize-20 white raleway-regular">Sed ut perspiciatis unde omnis iste natus</p>
+              <p class="blog-high-comment raleway-light white">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </br>
```
Comment (rogue707Dev) [inline src/index.html]: Put line breaks inside p tag

File: src/assets/css/style.scss (hunk 4 above)
Comment (lazydeveloperme) [reply src/assets/css/style.scss]: Better to separate into font-family, font-weight and font-style

**Pull request closed without merging**